platform-tests/example-apps/create-account-holding-page/nginx.conf

```diff
@@ -0,0 +1,54 @@
+worker_processes  1;
+daemon off;
+
+error_log <%= ENV.fetch("APP_ROOT") %>/nginx/logs/error.log;
+events {
+  worker_connections 1024;
+}
+
+http {
+  charset utf-8;
+  log_format access_json '{"logType": "nginx-access", '
+                         ' "remoteHost": "$remote_addr", '
+                         ' "user": "$remote_user", '
+                         ' "time": "$time_local", '
+                         ' "request": "$request", '
+                         ' "status": $status, '
+                         ' "size": $body_bytes_sent, '
+                         ' "referer": "$http_referer", '
+                         ' "userAgent": "$http_user_agent", '
+                         ' "requestTime": $request_time, '
+                         ' "httpHost": "$http_host"}';
+
+  access_log <%= ENV.fetch("APP_ROOT") %>/nginx/logs/access.log.json access_json;
+
+  upstream static_backend {
+    server localhost:8989;
+  }
+
+  server {
+    listen <%= ENV.fetch("PORT") %>;
+    server_name localhost;
+
+    location / {
+      root <%= ENV["APP_ROOT"] %>/public;
+      try_files $uri /index.html;
+    }
+
+    error_page 405 =200 @405;
```

Comment:
I chose to redirect to `GET` and provide a response body to make life easier for me as a tester. I can see in a browser the correct response. I gave little care to whether it was appropriate because this is an endpoint not advertised to tenants, shouldn't be used, and will only ever be seen by us or an attacker.

I can change it if you have strong feelings, but I don't think it is a good use of our time.

Reply:
The main reason is that this way if something happens to call it, it would be more likely to fail properly. With a 200 we risk of having a parsing error somewhere and having issues troubleshooting.  

It is not a big change, but no strong feelings from me. If you have strong ones to keep it like this, I am OK with it :-D

Reply:
Understandable. I think we should keep it as is for now. This whole approach to route blocking is a temporary solution anyway.